script/reset-environment.sh

```diff
@@ -1,10 +1,10 @@
 #!/usr/bin/env bash
 if ! [ -x "$(command -v yarn)" ]; then
     echo "Installing yarn..."
-    npm i -g yarn@1.12.3
+    npm i -g yarn@1.21.1
```

Comment:
This was missed during the Yarn update and I just caught it while searching our Yarn installs.